src/Microsoft.DotNet.Core.Build.Tasks/GenerateDepsFile.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.DependencyModel;
+using Newtonsoft.Json;
+using NuGet.Frameworks;
+using NuGet.ProjectModel;
+
+namespace Microsoft.DotNet.Core.Build.Tasks
+{
+    /// <summary>
+    /// Generates the $(project).deps.json file.
```

Comment:
nit: this comment confuses me some because this makes me think that the deps file will always be $(AssemblyName).deps.json, but is configurable separately through the DepsFilePath Argument